manifests/j/JAMSoftware/TreeSize/Free/4.6.1/JAMSoftware.TreeSize.Free.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=QUSU.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: JAMSoftware.TreeSize.Free
+PackageVersion: 4.6.1
```

Comment:
```suggestion
PackageVersion: 4.6
```
I'm still seeing `4.6` in Control Panel:
![image](https://user-images.githubusercontent.com/11600822/191376261-8f300bc6-d285-4925-a69a-f307ebc99921.png)